### PR TITLE
Challenge/irene-perez-todolist-redux

### PR DESCRIPTION
**Pull request by IrenePerezAlonso**

### CODE REVIEW

File: irene-perez/challenge/todolist-redux/todo-list/src/components/TaskList.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import TASKS from '../constant/task-mock';
+
+function TaskList() {
+  return (
+    <ul>
+      {TASKS.map((task) => (
+        <li>
```
Comment (shd8) [inline irene-perez/challenge/todolist-redux/todo-list/src/components/TaskList.jsx:10]: When you do a loop to render components you need to provide a key. In this case you should assign a key for evert <li>, for example: <li. className = {task.id} >

**Review by shd8 — APPROVED** [review]
Good practice, look at the comment, you have little things to repair at the stucture.

File: irene-perez/challenge/todolist-redux/todo-list/src/components/TaskList.jsx (hunk 1 above)
Comment (edalandete) [reply irene-perez/challenge/todolist-redux/todo-list/src/components/TaskList.jsx:10]: this should be <li. **key** = {task.id} >

File: irene-perez/challenge/todolist-redux/todo-list/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (edalandete) [inline irene-perez/challenge/todolist-redux/todo-list/public/index.html:27]: remember to change the title of yout application

File: irene-perez/challenge/todolist-redux/todo-list/src/constant/task-mock.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const TASKS = [
+  { id: 1, text: 'Comprar leche' },
+  { id: 2, text: 'Sacar al perro' },
+  { id: 3, text: 'Comprar medias' },
+  { id: 4, text: 'Ir al banco' },
+  { id: 6, text: 'Atracar banco' }
```
Comment (edalandete) [inline irene-perez/challenge/todolist-redux/todo-list/src/constant/task-mock.js:6]: jajajajajajajajajajajajajaj

**Review by edalandete — APPROVED** [review]
Me encanta!!

File: irene-perez/challenge/todolist-redux/todo-list/src/components/TaskList.jsx (hunk 1 above)
Comment (IrenePerezAlonso) [reply irene-perez/challenge/todolist-redux/todo-list/src/components/TaskList.jsx:10]: okey :)

**Review by IsabelGomez87 — APPROVED** [review]

**Review by Emerson-feral — APPROVED** [review]

**Pull request merged**